www/_posts/2020-10-02-cordova-electron-release-2.0.0.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Electron 2.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Electron 2.0.0`!  This is one of Cordova's supported platforms for building Electron applications.
+
+* [cordova-electron@2.0.0](https://www.npmjs.com/package/cordova-electron)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove electron
+cordova platform add electron@2.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add electron@2.0.0
+```
+
+Some of the notable breaking changes & new features in this release are:
+
+* The **Electron** core dependencies have been updated.
+  * **Electron**: 10.1.2
+  * **Chromium**: 85.0.4183.98
+  * **Node**: v12.16.3
+  * **V8**: v8.5
+* **NodeJS** 6 and 8 is no longer supported. This release requires the development environment to have **NodeJS** 10.x or higher. It is recommended to use the current LTS, which is `12.18.4` at the time of this release.
+* DevTool extensions can now be added to debug builds to improve debugging capabilities. [GH-160](https://github.com/apache/cordova-electron/pull/160)
+* Support for using the custom `scheme` and `hostname` `preference` flags is now available. It can be easily configured in your Cordova project by setting the preference options `scheme` and `hostname` in the `config.xml` file.
+
+    ```xml
+    <preference name="scheme" value="app" />
+    <preference name="hostname" value="localhost" />
+    ```
+
+* Support passing of **Electron** arguments to the `cordova run` command. This is useful to pass the inspect flag to debug the main process.
+
+For a quick start guide and in-depth configuration setup, please check out our [Cordova Electron Documentation](https://github.com/apache/cordova-electron/blob/rel/2.0.0/DOCUMENTATION.md)!
```

Comment:
Is this going to the main docs of cordova.apache.org? I don't know how docs work.

If yes we should link there instead of the file on Github.

Reply:
will be eventually but the website is not updated.

For example: 

* There is no Cordova 10.x docs to point to yet. It was not released.
* Pointing to Cordova 9.x docs would be wrong as it was related to the Cordova Electron 1.x releases.
* Pointing to dev would work maybe today, but tomorrow it could change and include major information.

Since Cordova-Electron 2.x is already released to npm, I think it is best to leave it as is so we can get the release announcement out.

Then we can look into releasing Cordova CLI 10.x docs and maybe update the link later if needed... Just not delay an announcement for a release that was already released 6 hours ago.